src/ontology/mondo-edit.obo

```diff
@@ -38496,27 +38496,42 @@ is_a: MONDO:0020115 {source="MONDO:Redundant", source="NCIT:C35434"} ! secondary
 
 [Term]
 id: MONDO:0002441
-name: Jervell-Lange Nielsen syndrome
+name: Jervell and Lange-Nielsen syndrome
 def: "An autosomal recessive inherited syndrome caused by mutations in the KCNE1 and KCNQ1 genes. It is characterized by congenital hearing loss and arrhythmia. It is a form of long QT syndrome." [NCIT:P378]
 subset: gard_rare
+subset: ordo_clinical_subtype {source="Orphanet:90647"}
+subset: prototype_pattern
 synonym: "Cardioauditory syndrome of Jervell and Lange-Nielsen" RELATED [GARD:0003048]
 synonym: "deafness, congenital, and functional heart disease" RELATED [GARD:0003048]
+synonym: "Jervell and Lange Nielsen syndrome" EXACT [NCIT:C84793]
+synonym: "Jervell and Lange-Nielsen syndrome 1" RELATED [MONDO:Lexical, OMIM:220400]
+synonym: "Jervell and Lange-Nielsen syndrome type 1" RELATED [MONDORULE:1, OMIM:220400]
 synonym: "Jervell and Lange-Nielson syndrome" EXACT [DOID:2842]
 synonym: "Jervell Lange-Nielsen syndrome" RELATED [GARD:0003048]
-synonym: "JLNS1" RELATED ABBREVIATION [GARD:0003048]
+synonym: "Jervell-Lange Nielsen syndrome" EXACT [DOID:2842]
+synonym: "JLNS1" RELATED ABBREVIATION [GARD:0003048, OMIM:220400]
+synonym: "long QT interval-deafness syndrome" EXACT [Orphanet:90647]
 synonym: "prolonged QT interval in EKG and sudden death" RELATED [GARD:0003048]
 synonym: "Surdo-cardiac syndrome" RELATED [GARD:0003048]
 xref: DOID:2842 {source="MONDO:equivalentTo"}
 xref: GARD:0003048 {source="MONDO:equivalentTo"}
+xref: ICD10:I45.8 {source="ORDO:90647/attributed", source="ORDO:90647/ntbt", source="Orphanet:90647"}
+xref: MedDRA:10057936 {source="ORDO:90647/e", source="Orphanet:90647"}
 xref: MESH:D029593 {source="DOID:2842", source="MONDO:equivalentTo"}
-xref: NCIT:C84793 {source="DOID:2842", source="MONDO:superClassOf"}
-xref: OMIM:220400 {source="GARD:0003048", source="DOID:2842", source="MONDO:superClassOf"}
-xref: OMIM:612347 {source="GARD:0003048", source="DOID:2842", source="MONDO:superClassOf"}
-xref: Orphanet:90647 {source="GARD:0003048", source="MONDO:superClassOf"}
-xref: SCTID:373905003 {source="DOID:2842", source="MONDO:superClassOf"}
+xref: NCIT:C84793 {source="MONDO:equivalentTo"}
+xref: OMIM:220400 {source="GARD:0003048", source="DOID:2842", source="MONDO:superClassOf", http://purl.obolibrary.org/obo/mondo#source="Orphanet:90647"}
+xref: OMIM:612347 {source="GARD:0003048", source="DOID:2842", source="MONDO:superClassOf", http://purl.obolibrary.org/obo/mondo#source="Orphanet:90647"}
+xref: OMIMPS:220400 {source="MONDO:cjm", source="MONDO:equivalentTo"}
+xref: Orphanet:768 {source="MONDO:subClassOf", source="OMIM:220400"}
+xref: Orphanet:90647 {http://purl.obolibrary.org/obo/mondo#source="GARD:0003048", source="MONDO:equivalentTo", source="OMIM:220400"}
```

Comment:
same source issue as above

when you hover over the annotations in Protege, you can see where they come from.

![image](https://user-images.githubusercontent.com/6722114/126412690-0b715062-83f8-4244-85bc-716dec4fe887.png)